code/game/turfs/simulated/floor/fancy_floor.dm

```diff
@@ -113,6 +114,32 @@
 		icon_state = "basalt[rand(0, 12)]"
 		set_basalt_light(src)
 
+
+/turf/open/floor/grass/fakebasalt //Heart is not a real planeteer power
+	name = "aesthetic volcanic flooring"
+	desc = "Safely recreated turf for your hellplanet-scaping"
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "basalt"
+	floor_tile = /obj/item/stack/tile/basalt
+	ore_type = /obj/item/weapon/ore/glass/basalt
+	slowdown = 0
+
+/turf/open/floor/grass/fakebasalt/Initialize()
+	..()
+	if(prob(15))
+		icon_state = "basalt[rand(0, 12)]"
+		set_basalt_light(src)
+
+/turf/open/floor/grass/fakebasalt/attackby(obj/item/C, mob/user, params)
```

Comment:
Just replace the "uproot" part with a var named something like 'turfverb' and use it on to give each turf a more appropriate description.